src/test/resources/TestNG.xml

```diff
@@ -5,9 +5,15 @@
 
     <test name="Example Test" preserve-order="false">
         <classes>
+ Homework
+            <class name="LoginTests"/>
+            <class name="Homework16"/>
+            <class name="BaseTest"/>
+
             <class name="Homework17" />
 <!--            <class name="ProfileTests"/>-->
 <!--            <class name="BaseTest"/>-->
+main
         </classes>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/211497229-44b015e7-a9b4-4ace-91ff-7350675af5e9.png)

For example

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG">
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="LoginTests"/>
            <class name="Homework16"/>
            <class name="BaseTest"/>
            <class name="Homework17" />
        </classes>
    </test>
</suite>
```